README.md

```diff
@@ -43,6 +43,10 @@ enabled = false
 ; path of the database
 path = ~/downloads/pirate-get/db
 
+[Search]
+; how many links to return after fetching all results
```

Comment:
What about "maximum number of results to show"?

Reply:
Done 👍🏻